src/control-plane/backend/lambda/api/service/quicksight/reporting-utils.ts

```diff
@@ -719,9 +753,21 @@ export function getFunnelTableVisualRelatedDefs(viewName: string, colNames: stri
   return columnConfigurations;
 }
 
-export function getEventLineChartVisualDef(visualId: string, viewName: string, titleProps: DashboardTitleProps, groupColumn: string) : Visual {
+export function getEventChartVisualDef(visualId: string, viewName: string, titleProps: DashboardTitleProps,
+  quickSightChartType: QuickSightChartType, groupColumn: string) : Visual {
 
-  const visualDef = readFileSync(join(__dirname, './templates/event-line-chart.json'), 'utf8');
+  let templatePath = '';
+  if (quickSightChartType === QuickSightChartType.LINE) {
```

Comment:
use `switch` is better

Reply:
After solving the above problem, we no longer need to use the `switch`